test/io/test_io.cpp

```diff
@@ -1365,6 +1366,36 @@ TEST (PCL, Locale)
 #endif
 }
 
+//////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+TEST (PCL, AutoIO)
+{
+  PointCloud<pcl::PointXYZ> cloud;
+  PointCloud<pcl::PointXYZ> cloud_pcd;
+  PointCloud<pcl::PointXYZ> cloud_ply;
+  PointCloud<pcl::PointXYZ> cloud_ifs;
+
+  cloud.width  = 640;
+  cloud.height = 480;
+  cloud.resize (cloud.width * cloud.height);
+  cloud.is_dense = true;
+
+  save ("test_autoio.pcd", cloud);
+  save ("test_autoio.ply", cloud);
+  save ("test_autoio.ifs", cloud);
+
```

Comment:
I'm not really fond of dumping stuff into the disk and not cleaning up after. Can you erase these files upon exiting the test?